addons/hr_homeworking/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+from odoo.tools.misc import DEFAULT_SERVER_DATE_FORMAT
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def get_worklocation(self, start_date, end_date):
+        employee_id = self.env['hr.employee'].search([
+            ('user_id.id','=', self.env.user.id),
+            ('company_id.id','=',self.env.company.id)])
+        return employee_id.get_worklocation(start_date,end_date)
```

Comment:
I do that because the link between work_location and hr_homworking is hr_employee. And with the calendar, I have a res_partner, so the res_partner call the function in hr_employee :)